controller/graphql/schema.go

```diff
@@ -1,253 +1,36 @@
-// Code generated by github.com/ipld/go-ipld-prime/schema/gen/graphql/server, DO NOT EDIT.
```

Comment:
are you no longer letting code-gen generate this?

Reply:
because the generated schema is based on .schema file and the generated go files. But bindnode need not the generated go files

Reply:
but this file is generating code using https://github.com/ipld/go-ipld-graphql - the generated is automatically created for graphql stubs based on the ipld schema.

you should be able to pull out a type system from the bind-node schema and use that to generate this file, or you can remove it fully.
hand-coding it is probably not great for maintainability

Reply:
ok, I will remove it